Cait's Unity Test/Assets/Velocity.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Velocity : MonoBehaviour
+{
+    public Vector2 vel;
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update() => gameObject.transform.Translate(vel * Time.deltaTime);
```

Comment:
```csharp
[SerializeField] private Vector2 vel;

void Update() => gameObject.transform.Translate(vel * Time.deltaTime);
```

Reply:
no need for changing the backing vector visibility; there are a million reasons you may want to read and/or write an object's velocity from elsewhere and there are no invariants that could be broken by random places writing to it, as there are no invariants to be broken